### PR TITLE
Added 8.x compatibility

### PR DESCRIPTION
**Pull request by sbruggmann**
Increased supported Neos version, updated @neos-project dependencies and added missing regenerator-runtime dependency.

It would make sense to create a new major version 1.0 and limit the supported Neos versions to 8.x

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "name": "flowpack/restrictcreation",
     "license": "MIT",
     "require": {
-        "neos/neos": "^3.0|^4.0|^5.0|^7.0"
+        "neos/neos": "^3.0|^4.0|^5.0|^7.0|^8.0"
```
Comment (bwaidelich) [inline composer.json:7]: This should be changed to
```suggestion
        "neos/neos": "^8.0"
```
I suppose?

**Comment by bwaidelich** [comment]
@sbruggmann Thanks for this. I can take care of a new major release, but IMO the version constraints need to be adjusted, right?

**Comment by bwaidelich** [comment]
I took the freedom to adjust the composer dependencies as suggested and tagged a 1.0.0 with the changes: https://github.com/Flowpack/restrict-creation/tree/1.0.0

**Comment by sbruggmann** [comment]
@bwaidelich Thank you for the update!
And **sorry** for ghosting 🙈

**Pull request merged**